<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="ATMECSINDT-080" failures="1" tests="3" name="Test validate ie" time="24.193" errors="0" timestamp="05 Nov 2019 06:28:01 GMT">
  <testcase classname="com.atmecs.practo.testbase.InvokeBrowser" name="setup" time="21.35"/>
  <testcase classname="com.atmecs.practo.testscripts.AddToCart" name="clickOnDiagnostics" time="2.839"/>
  <testcase classname="com.atmecs.practo.testscripts.AddToCart" name="selectCity" time="0.001">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at java.base/java.io.File.<init>(File.java:276)
at com.atmecs.practo.utils.ReadXML.readXmlFile(ReadXML.java:20)
at com.atmecs.practo.testscripts.AddToCart.selectCity(AddToCart.java:56)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:835)
]]>
    </failure>
  </testcase> <!-- selectCity -->
</testsuite> <!-- Test validate ie -->
